vsphere/panels.libsonnet

```diff
@@ -0,0 +1,1118 @@
+            + table.standardOptions.withDecimals(1)
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Memory ballooned')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('mbytes')
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Memory swapped')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('mbytes')
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Disk throughput')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('KiBs')
+          ),
+        ])
+        +
+        table.queryOptions.withTransformationsMixin([
+          {
+            id: 'joinByField',
+            options: {
+              byField: 'vcenter_host_name',
+              mode: 'outer',
+            },
+          },
+          {
+            id: 'filterFieldsByName',
+            options: {
+              byVariable: false,
+              include: {
+                names: [
+                  'vcenter_host_name',
+                  'vcenter_vm_name 1',
+                  'Value #A',
+                  'Value #C',
+                  'Value #D',
+                  'Value #B',
+                  'Value #E',
+                ],
+              },
+            },
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              includeByName: {},
+              indexByName: {
+                'Value #A': 2,
+                'Value #B': 6,
+                'Value #C': 3,
+                'Value #D': 4,
+                'Value #E': 5,
+                vcenter_host_name: 0,
+                'vcenter_vm_name 1': 1,
+              },
+              renameByName: {
+                'Value #A': 'CPU utilization',
+                'Value #B': 'Disk throughput',
+                'Value #C': 'Memory utilization',
+                'Value #D': 'Memory ballooned',
+                'Value #E': 'Memory swapped',
+                'Value #F': 'Memory swapped',
+                vcenter_cluster_name: 'Cluster',
+                'vcenter_cluster_name 1': 'Cluster',
+                vcenter_host_name: 'Host',
+                'vcenter_host_name 1': 'Host',
+                vcenter_vm_name: 'Name',
+                'vcenter_vm_name 1': 'Name',
+                'vcenter_vm_name 2': '',
+              },
+            },
+          },
+        ]),
+
+      disksTable:
+        commonlib.panels.generic.table.base.new(
+          'Disks table',
```

Comment:
If there are only 2 columns for this table, choosing a different alignment would look better to eliminate all the space in the middle.

Reply:
Was waiting on an extra metric to be added to vcenter receiver, has been updated!